### DISCUSSION TITLE
IOS Simulator not working on my Expo React Native project.

### DISCUSSION THREAD

**Discussion by michelleabsantos** [NativeScript/NativeScript#10306] [Q&A]
Hello! I need help with Expo IOS Simulator.
I am developing an app using Expo React Native in VSCode, and suddenly, for 4 days, I've been having this message popping up whenever I try to run the project in the simulator.

![screenshot_2023-06-11_at_3 50 15_pm](https://github.com/NativeScript/NativeScript/assets/110417316/23d3859f-1c03-4918-ad8c-9e037071a2a5)

I already uninstalled and installed XCode, cleaned the simulator cache, uninstalled and installed package-lock.json, restarted my computer (50x), moved simulator to the thrash and brought it back, not to mention other 30 different things I tried following instructions from different support channels... and nothing... 

Can someone help me troubleshoot this? Thanks!!!

**Comment by NathanWalker** [comment]
Hi @michellebessa sorry to hear of the trouble, you will get accurate help with the Expo team here:
https://github.com/expo/expo/discussions

This discussion board is for [NativeScript](https://nativescript.org/)
